project/templates/site_client_to_services_binder/show.html.twig

```diff
@@ -0,0 +1,22 @@
+{% extends 'base.html.twig' %}
```

Comment:
use ful to keep all the file for SiteClientToServicesBinder in view ad also the form ?